src/backup.ts

```diff
@@ -243,6 +253,45 @@ Please use the format 'my-backup' or '${cluster.name}/backups/my-backup'.`);
     });
   }
 
+  /**
+   * Copy this backup
```

Comment:
"Copy this backup..." to where? please add to the documentation

Reply:
I added a paragraph here. Some functions in the client library have a long explanation of what they do and some don't, but I extended this one so its purpose is more clear.